test/unittests/model/versionrepr_test.py

```diff
@@ -9,12 +9,23 @@ class TestVersionRepr:
     def test_text(self):
         v1 = Version("master+build2")
         vr = _VersionRepr(v1)
-        assert vr.major() == "master"
-        assert vr.minor() == "master"
-        assert vr.patch() == "master"
-        assert vr.pre() == "master"
+        assert vr.major() == "master.Y.Z"
+        assert vr.minor() == "master.0.Z"
+        assert vr.patch() == "master.0.0"
```

Comment:
Maybe this is the issue, that a package called only ``mypkg/master`` will be adding extra digits?
What will happen to ``opengl/system`` as a dependency? It will probably break other consumers ``package_ids``?
This might be a different use case that the ``cci.2.3`` version